styleguide/source/assets/js/subcategory-exploration.js

```diff
@@ -1,4 +1,45 @@
-$('.ama__subcategory-exploration__show-more').click(function() {
-  $('.ama__subcategory-exploration__list').toggleClass('ama__subcategory-exploration__list--expanded');
-  $(this).toggleClass('ama__subcategory-exploration__show-more--expanded');
-});
+/**
+ * @file
+ * Subcategory
+ *
+ * JavaScript should be made compatible with libraries other than jQuery by
+ * wrapping it with an "anonymous closure". See:
+ * - https://drupal.org/node/1446420
+ * - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
+ */
+(function ($, Drupal) {
+  Drupal.behaviors.subcategoriesExploration = {
+    attach: function(context, settings) {
+
+      var subcategoryListContainer = $('.ama__subcategory-exploration__list');
```

Comment:
Read https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards

**In case a variable contains a jQuery object, the variable MUST start with a dollar sign ($):**